core/src/main/java/org/apache/iceberg/view/ViewMetadata.java

```diff
@@ -78,6 +79,9 @@ default Integer currentSchemaId() {
 
   List<MetadataUpdate> changes();
 
+  @Nullable
+  String metadataFileLocation();
```

Comment:
Looks like this needs to be rebased on #8608 that just went in.

Reply:
correct. I've rebased the PR now, so those changes shouldn't be visible here anymore